### ISSUE TITLE
Collision - Convert convex to box shape

### ISSUE THREAD

**Issue by neomonkeus** [niftools/blender_niftools_addon#182]
Migrated from sf.net

I asked you via PM about the possibility of a certain script, to overcome the box shape collision bounding box issue (Blender), for PYFFI. It's a type of conversion script to convert perfectly
cubic/rectangular shaped convex collisions, into box shape collision (with bhkTransformShapes for each box shape, to account for any Translation or rotation offset). I think box shape collisions are more performance friendly than box-shaped convex collision, tbh. I also think I read something like that on Saidenstorm's collision tutorial, on his personal website.

You mentioned it might require something called "numpy", to do the required math calculations. You also said this "numpy" might also have to be downloaded by those of us who want to use such a PYFFI script, and thus add another program dependency. Is it then possible, to make the use of such a conversion script and program needed, optional? Or can "numpy" be somehow packaged with PYFFI in same download and install, to make things easier on modders?

Koniption

**Comment by HENDRIX-ZT2** [comment]
I think this is at least a new feature suggestion, but it is probably too broad and not really possible. Seems like a highly complicated heuristic, too. The plugin should not have to worry about this.
I'd say: close off as invalid.

**Comment by neomonkeus** [comment]
Yeah that is why I pushed it out to the 3.0.0 new feature milestone. 
Some of this is done within the plugin already to convert some shapes like capsule > sphere.
